sdk/go/common/util/cmdutil/args.go

```diff
@@ -29,7 +29,7 @@ func ArgsFunc(argsValidator cobra.PositionalArgs) cobra.PositionalArgs {
 		if err != nil {
 			return errors.Join(cmd.Help(), err)
 		}
-		return err
+		return nil
```

Comment:
Make it a bit more clear at a glance this is returning nil, we check for err != nil above